bin/check_tron_jobs.py

```diff
@@ -188,7 +206,9 @@ def check_job_result(job, client, dry_run):
         log.info("Would have sent this event to sensu: ")
         log.info(result)
     else:
-        log.debug("Sending event: {}".format(result))
+        for key in SENSU_REQUIRED_KEYS:
```

Comment:
Leave the debug in here. That is how we found it in the first place.

There is no need to iterate over these keys. The only keys we need to worry about being unset are runbook and team. Please replace those in the dictionary as needed.

For runbook please say something like "No runbook specified. Please add one with 'runbook: foo'"